src/Transform/feature2Mesh.js

```diff
@@ -415,28 +407,31 @@ function featureCollectionToThree(featureCollection, options) {
 }
 
 /**
- * @module Feature2Mesh
+ * Converts [Features]{@link module:GeoJsonParser} to [THREE.Mesh]{@link
+ * https://threejs.org/docs/#api/objects/Mesh}.
+ *
+ * @function feature2Mesh
+ *
+ * @param {module:GeoJsonParser~Feature} feature - a Feature or an array of
+ * Feature.
+ * @param {Object} [options] - Options controlling the conversion.
+ * @param {number|function} options.altitude - Define the base altitude of
+ * the mesh.
+ * @param {number|function} options.extrude - If defined, polygons will be
+ * extruded by the specified amount.
+ * @param {Object|function} options.color - Define per feature color.
+ *
+ * @return {function} Returns a [THREE.Mesh]{@link
+ * https://threejs.org/docs/#api/objects/Mesh}. If an array of Feature is
+ * specified, it will be converted to a [THREE.Group]{@link
+ * https://threejs.org/docs/#api/objects/Group}.
  */
-export default {
-    /**
-     * Return a function that converts [Features]{@link module:GeoJsonParser} to Meshes. Feature collection will be converted to a
-     * a THREE.Group.
-     *
-     * @param {Object} options - options controlling the conversion
-     * @param {number|function} options.altitude - define the base altitude of the mesh
-     * @param {number|function} options.extrude - if defined, polygons will be extruded by the specified amount
-     * @param {object|function} options.color - define per feature color
-     * @return {function}
-     */
-    convert(options = {}) {
-        return function _convert(feature) {
-            if (!feature) return;
-
-            if (Array.isArray(feature)) {
-                return featureCollectionToThree(feature, options);
-            } else {
-                return featureToThree(feature, options);
-            }
-        };
-    },
-};
+export default function (feature, options = {}) {
```

Comment:
Let's rename `options` to `style` then.

Reply:
(and maybe also support functions, though maybe not in this PR)

Reply:
Yep, that would be better.

Reply:
In fact I prefer to let the `options` naming. It makes sense to change it in `feature2Mesh`, but not in `feature2Texture`. I feel more that I should change the `style` options when adding a layer. Any suggestion on a name ?

Reply:
Yes indeed.

We might need something more generic: `paint` like mapboxgl? Or `draw`? `drawOptions`?

@peppsac, you said you wanted to stay with `style` though?

Reply:
I don't mind `options` here (but I prefer `style` when its declared within the layer)